src/frontend/js/settings.ts

```diff
@@ -1,15 +1,13 @@
-import { modelName } from './types/models';
-
-export const API_ENDPOINTS = {
+const API_ENDPOINTS = {
```

Comment:
Why are you keeping this variable ? It is not exported and not used at all. You should maybe remove it ?

Reply:
Good catch!